java-agent/instrumentation/spring-webmvc-3.1/src/main/java/io/opentelemetry/auto/instrumentation/springweb/DispatcherServletInstrumentation.java

```diff
@@ -92,12 +92,13 @@ private void muzzleCheck(final HandlerMethod method) {
   public static class ErrorHandlerAdvice {
     @Advice.OnMethodEnter(suppress = Throwable.class)
     public static void nameResource(@Advice.Argument(3) final Exception exception) {
-      final AgentSpan span = activeSpan();
+      final Span span = TRACER.getCurrentSpan();
       if (span != null && exception != null) {
         DECORATE.onError(span, exception);
         // We want to capture the stacktrace, but that doesn't mean it should be an error.
         // We rely on a decorator to set the error state based on response code. (5xx -> error)
-        span.setError(false);
+        // span.setError(false); // TODO: Is this needed? Doesn't seem to be a similar method in
+        // OpenTelemetry!
```

Comment:
```
// We want to capture the stacktrace, but that doesn't mean it should be an error.
// We rely on a decorator to set the error state based on response code. (5xx -> error)
BaseDecorator.addThrowable(span, exception);
```
This will track the exception without calling setError